src/store/slices/auth/actionsAsync.ts

```diff
@@ -1,11 +1,12 @@
 import { authActions } from './index';
-import { api } from 'hopr-sdk';
+import { api } from '@hoprnet/hopr-sdk';
+const { getInfo } = api;
 
 export const login = (loginData: any) => {
```

Comment:
this should receive a type and I think it would be something along the lines of {apiToken: string; apiEndpoint: string}

Reply:
Good point, changed